### PR TITLE
Support for Business Removal in SJT Transfer

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#18066
/bcgov/entity#18656
/bcgov/entity#18197

*Description of changes:*
* New component for handling Business Removal in Transfers to Surviving Joint Tenants.
* Updated labels, messaging and validations to handle 'Historical' business removals alongside 'Deceased' owner removals in Transfer to SJT's.
* Minor Label fix edge case scenario (ie, ADDED/CHANGED badge when group has removed all original owners) see 18197
* Minor property updates and handling to submit Transfer SJT's when a business is present.
* Misc styling fixes to correct the spacing of Death Certificates and the new component in the Home Owners table during a transfer.
* Some pics for context, for groupless and grouped scenarios:

Screenshot 2024-01-17 at 1 01 10 PM

Screenshot 2024-01-17 at 1 01 24 PM

Screenshot 2024-01-17 at 1 03 23 PM

Screenshot 2024-01-17 at 1 09 47 PM

Screenshot 2024-01-17 at 1 10 05 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrTransfers/DeathCertificate.vue [hunk 1]
```diff
@@ -119,12 +122,6 @@ export default defineComponent({
     const { setUnsavedChanges } = useStore()
     const deathCertificateForm: FormIF = ref(null)
     const deathCertificateNumberRef: FormIF = ref(null)
-    const deathCertificateNumberRules = computed(
-      (): Array<()=>string|boolean> => customRules(
-        maxLength(20),
-        required('Enter Death Certificate Registration Number')
-      )
-    )
```
Comment (cameron-eyds) [inline ppr-ui/src/components/mhrTransfers/DeathCertificate.vue:127]: Duplicate definition, was already defined in localState as a computed value

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1693-80g5x92w.web.app

**Comment by cameron-eyds** [comment]
@doug-lovett I completed a few of these filings and just wanted to confirm something about the output content. 
Are they supposed to display the removed owners or typically just the new/current set of owners?

The outputs looked good otherwise!

@cameron-eyds for transfers the output content has not changed in quite some time - just the new/current set of owners.

- Sounds good sir, thank you verifying!

**Review by doug-lovett — APPROVED** [review]
Very nice.

**Review by dimak1 — APPROVED** [review]
Looks good.

**Pull request merged**